Python/pywarpx/Diagnostics.py

```diff
@@ -7,6 +7,7 @@
 from .Bucket import Bucket
 
 diagnostics = Bucket('diagnostics', _diagnostics_dict={})
+reduced_diagnostics = Bucket('warpx', _diagnostics_dict={})
```

Comment:
I notice the inconsistency in the instance name argument, but this comes from the naming used in the C++ parm parsing, using `diagnostics.diag_names` and `warpx.reduced_diag_names`. I don't know which prefix is better, but it would be good to use the same prefix (though probably should be a different PR).

Reply:
I agree using the diagnostic prefix for both would be nicer, but, as you said, probably a follow-up PR.